src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -378,8 +378,20 @@ public void storageIntroInformation() {
     }
 
     boolean usesExternalStorage =
-        dmp.getHostList().stream().anyMatch(ExternalStorage.class::isInstance);
-    boolean usesInternalStorage = dmp.getHostList().stream().anyMatch(Storage.class::isInstance);
+        dmp.getHostList().stream()
+            .filter(ExternalStorage.class::isInstance)
+            .map(ExternalStorage.class::cast)
+            .anyMatch(
+                storage ->
+                    storage.getIsManagedInternally() != null && !storage.getIsManagedInternally());
+
+    boolean usesInternalStorage =
+        dmp.getHostList().stream()
+            .anyMatch(
+                host ->
+                    (host instanceof Storage)
+                        || (host instanceof ExternalStorage
+                            && ((ExternalStorage) host).getIsManagedInternally()));
```

Comment:
This makes it a bit easier to parse :)
```suggestion
                    (host instanceof Storage) || (
                      host instanceof ExternalStorage && 
                      ((ExternalStorage) host).getIsManagedInternally())
                    );
```

Reply:
This is automatically done by spotless apply, otherwise the pipeline does not run through. From what I see it is anyways only visual?

Reply:
Yes, this is only visual, feel free to ignore.